frontend/src/views/ContactView.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <!-- header -->
+  <div class="contact-header">
+    <div class="contact-header-text">KONTAKT</div>
+  </div>
+
+  <!-- this is main container -->
+  <div class="contact-container">
+    <h2 class="contact-subheading">KONTAKTUJTE NÁS</h2>
+
+    <!-- contact cards -->
+    <div class="contact-cards">
+      <div v-for="contact in contacts" :key="contact.name" class="contact-card">
+        <img :src="contact.image" alt="Profile" class="contact-image" />
+        <h3 class="contact-name">{{ contact.name }}</h3>
+        <p class="contact-role">Organizátor</p>
+        <p class="contact-phone">{{ contact.phone }}</p>
+        <a :href="contact.email" class="contact-email">{{ contact.email }}</a>
+      </div>
+    </div>
+  </div>
+
+
+  <!-- Ak chceme používať Google Maps bez cookies (čo znamená, že nevyužívate dynamické zobrazenie mapy cez Google Maps JavaScript API),
+    problémom môže byť niekoľko vecí. 
```

Comment:
Myslím že existuje nejaký package na Google Maps API pre Vue (pozri REAME linky úplne dole). Ale to vyriešime potom v inom PR